server/models.py

```diff
@@ -313,10 +314,9 @@ class Assignment(Model):
     files = db.Column(JsonBlob)  # JSON object mapping filenames to contents
     course = db.relationship("Course", backref="assignments")
 
-
-    UserAssignment = namedtuple('UserAssignment',
-                                ['assignment', 'subm_time', 'group', 'final_subm'])
-
+    user_assignment = namedtuple('UserAssignment',
```

Comment:
Why this change?  `namedtuple` returns a class which are generally named CamelCase